Aben's and Pablo's worked merged in one.

abenezer-molla opened the pull request:
Separate Signup for Charity and Donor, along side with working login that will take us to the CharityDashBorad and DonorDashboard page that were build by Pablo. I have brought them together and done the routing to the right page along with the functioning auth! Everyone is welcome to review it (specially Pablo and Cyrus).

pabloguarneros commented:
Hey, @abenezer-molla. The implementation seems to be getting us away from the work the rest of the team has done. Although there's a choice to keep editing this branch, I'd rather we go full on the current version.

Reasons why the deviation is too great to consider merging:

- When checking out this branch, the previous backend is gone. 
- This is running on Python 3.9 while the previous implementation ran on 3.6. Thus, the dependencies will have to change.
- When running the app through python run.py, I hit the first error at import from main.py. 
-  The routes lack some of the depth that the previous ones went into.
-  I would recommend closing this request and not merging it. @abenezer-molla which parts of the code could we recover and integrate into the existing master branch without creating new folders, files, etc?

I could be wrong, so I would love to have @cyrushhc @Phil-Barv look into this too.

abenezer-molla commented:
@pabloguarneros No worries mate! We will move forward with things that will benefit the team at large. I will identify the codes that can be integrated into the existing master and let you know tomorrow night. 

Specifying the steps on how to run it should have been included in the READ.ME, and I can see that caused you a confusion. But I removed certain files and added others because non of the authentication was leading to the routing pages in all the previous versions of this app we worked on. But it worked on this version- trust me it was thought out and I wouldn't have restructured it as much if I didn't. 

Anyways, given the time we have, it makes sense to go with the working model in the "master" than to review this PR again and solve dependency issues. 

@cyrushhc  and @Phil-Barv - let's not waste time reviewing this PR. We will just  take a raw code from this and add it to the one in master without creating a new folder as Pablo mentioned! 

Good Night!

pabloguarneros commented:
Awesome! You're a gem and have a good night :)
